site/en/install/pip.html

```diff
@@ -241,14 +241,14 @@ <h2>3. Install the TensorFlow pip package</h2>
 <h3>Virtual environment install</h3>
 <pre class="devsite-terminal tfo-terminal-venv devsite-click-to-copy prettyprint lang-bsh">pip install --upgrade tensorflow</pre>
 <p>Verify the install:</p>
-<pre class="devsite-terminal tfo-terminal-venv devsite-click-to-copy prettyprint lang-bsh">python -c "import tensorflow as tf;print(tf.reduce_sum(tf.random.normal([1000, 1000])))"</pre>
+<pre class="devsite-terminal tfo-terminal-venv devsite-click-to-copy prettyprint lang-bsh">python -c "import tensorflow as tf; print(tf.constant('TensorFlow installed successfully'))"</pre>
```

Comment:
the computation was proposed by alex as a way to actually test that TF was installed, as opposed to some code paths which may shortcut around problems that show later. I think it would be better to add something like, "If a tensor is returned, you've installed TensorFlow successfully."

Also, please sign the CLA or set up your GitHub account to work with CLAbot :)

Reply:
Hey @lamberta!

> Also, please sign the CLA or set up your GitHub account to work with CLAbot :)

Working on getting it signed :)


> I think it would be better to add something like, "If a tensor is returned, you've installed TensorFlow successfully."

Would you suggest adding another print statement to the code or editing the success message in the following `<aside>` tag?